### PR TITLE
Change order of the TOC and improve events part

### PR DESCRIPTION
**Pull request by GlaceCoding**
Yesterday, I helped a student because she got a weird error on mlx_hook callback. I take some time before understand it was a prototype error on mousedown function. Then why I add `Prototype of event functions` part because there is no part in the doc explain this (I only know because I read the code).

And color should be after the getting start. Also the introduction in event, should be before hook because they use hook before understand events. 

EDIT: And I fix the id in hooking alias (in my last PR I invert the id).

### CODE REVIEW

File: libs/minilibx/events.md [hunk 1]
```diff
@@ -141,13 +141,29 @@ int	main(void)
 Here we register to the `KeyPress` event with the according `KeyPressMask`. Now
 whenever we press a key, the window will be closed.
 
+### Prototype of event functions
+
+Event functions have different prototype depending of the hooking event.
```
Comment (harm-smits) [inline libs/minilibx/events.md]: have `a` different prototype

File: libs/minilibx/events.md [hunk 2]
```diff
@@ -141,13 +141,29 @@ int	main(void)
...
+
+| Hooking event | code | Prototype            |
+| ------------- | :--: | -------------------- |
+| ON_KEYDOWN    |   2  | `keydown(int keycode, t_vars *vars)`   |
+| ON_KEYUP*     |   3  | `keyup(int keycode, t_vars *vars)` |
+| ON_MOUSEDOWN* |   4  | `mousedown(int keycode, int x, int y, t_vars *vars)` |
+| ON_MOUSEUP    |   5  | `mouseup(int keycode, int x, int y, t_vars *vars)` |
+| ON_MOUSEMOVE  |   6  | `mousemove(int x, int y, t_vars *vars)` |
+| ON_EXPOSE*    |  12  | `expose(t_vars *vars)` |
+| ON_DESTROY    |  17  | `close(t_vars *vars)` |
```
Comment (harm-smits) [inline libs/minilibx/events.md]: Can you sort this alphabetically and can you denote the prototypes as a function pointer?

**Review by harm-smits — CHANGES_REQUESTED** [review]
There actually is a mention of the event function signatures in the prototypes section, where it is defined as
```c
(*f)(int button, int x, int y, void *param);
```

Although, I indeed have not documented the `mlx_hook` function in the prototypes section. It would be cool if you could add it there as well.

File: libs/minilibx/events.md (hunk 2 above)
Comment (GlaceCoding) [reply libs/minilibx/events.md]: I ordered by id asc

**Comment by GlaceCoding** [comment]
I did my best, should I explain x_event argument or is it good ?

**Review by harm-smits — APPROVED** [review]

**Pull request merged**